### ISSUE TITLE
What happened to playlist playback functionality?

### ISSUE THREAD

**Issue by GeneralUser-NonDev** [clsid2/mpc-hc#1465]
Back in the ver 1.7.8 I have of MPC-HC I was able to create a playlist of playlists and it'd play it upon opening it from the OS GUI, I just discovered that in both 1.9.16 and 1.9.18 that isn't working anymore, I don't know about other releases. I know I, for one, would appreciate that being fixed. Thank You guys.

ex: (or at least similar)
---------.m3u file--------- (c\tvshow\tvshow_allseasons.m3u)
Season1\Season1.m3u (containing 20 episodes)
Season2\Season2.m3u (containing 20 episodes)
Season3\Season3.m3u (containing 20 episodes)
--------end .m3u file----------

note: normal playlists of .avi, .mkv, .mp3, etc all still seem to work fine, it's just the playlists of playlists that appears to be broken.

**Comment by clsid2** [comment]
Nested playlists are technically invalid for M3U.

I might add back this ability in the future, but it is super low priority.

**Comment by GeneralUser-NonDev** [comment]
I'm not sure how this works exactly but if/when you re-instate this function could you post in here so I get the notification? I've since reverted back to the 1.7.8 to have that function at the loss of the convenient save feature in the playlist in lieu of the "save as" as the only option, that and all my icons went screwy when I downgraded.

**Comment by slalaurette** [comment]
I agree that this was a very useful feature for multiple CD compilations, for example. I use them extensively and this regression is annoying. 

I understand that it's a low priority thing, though, and I want to point out that I'm extremely happy with MPC-HC and grateful with the developer, even if this doesn't get addressed. 

Thanks for bringing MPC-HC back!

**Comment by clsid2** [comment]
Should work again in latest development build.

**Comment by GeneralUser-NonDev** [comment]
I appreciate the heads up, however I tried it (on 1.9.23) and it didn't work, also noticed the "Save" option in the playlist is gone and only has "save as", Different issue but figured I'd mention it.

Would it matter if the m3u file(s) started with "#EXTM3U" (all caps) or not?

**Comment by clsid2** [comment]
1.9.23 is a release version not the development build (1.9.23.19).

It only works without #EXTM3U tags. So a simple list of m3u file names like in your first post.

**Comment by GeneralUser-NonDev** [comment]
Just tested the nightly one (accidentally grabbed 1.9.23.16 instead of 19, then noticed and tested again) and it worked with "Nested Playlist", Thank You. However I noticed it was still missing the "Save" option to go along with "Save as", again separate issue but figured I'd mention it in my Thank You post, now just for it to get to "release state".

Thank You.

![MPC-HC_1 9 23 16x64_M3Uissue](https://user-images.githubusercontent.com/96662916/198900971-23a3a4a6-558d-4943-aa32-cc2f0f529cac.JPG)
*note: Removed "#EXTM3U" tags from playlist(s) before testing new release.
*note2: My Norton quarantined the exe file, had to un-quarantine it, just FYI

Thank You for putting up with me, as my user name suggests, I know nothing about this coding stuff so I appreciate the patience and work done to bring back (and improve) this program.
